src/test/clojure/xtdb/information_schema_test.clj

```diff
@@ -536,3 +544,12 @@
              (count results)))
     (t/is (= types
              results))))
+
+(deftest schema-for-meta-tables-3550
+  (t/is (= [{:table-catalog "xtdb",
+             :table-schema "information_schema",
+             :table-name "tables",
+             :table-type "BASE TABLE"}]
```

Comment:
The table type in Postgres is a `VIEW`, some others are basic `BASE TABLE`'s. I don't know if we want to emulate the exact mapping or if `VIEW` and `BASE TABLE` should reflect our own internal structure. Everything a `VIEW` apart from `pg_user`.